freeipa/src/test/java/com/sequenceiq/freeipa/service/stack/ClusterProxyServiceTest.java

```diff
@@ -125,11 +135,11 @@ public void testUpdateClusterProxyRegisterationWhenCCMV2OrJumpgate(Tunnel ccmv2M
         freeIpa.setDomain("test.freeipa.domain");
```

Comment:
Remark: I think this test is a bit big, using 9 injections. Also, registerFreeipa method in the productive code is a bit long. It might make sense to think the code can be structured into more classes. I know cloudbreak is full of such methods, but maybe next time when this class is touched a refactoring could also be made.

Method name contains a typo (Registeration).

Reply:
agree